frontends/verilog/verilog_parser.y

```diff
@@ -702,10 +711,17 @@ opt_wire_type_token:
 
 wire_type_token:
 	hierarchical_type_id {
```

Comment:
With these changes, shouldn't `hierarchical_type_id` simply be one of the `logic_type`s?

Reply:
Yes it should! Fixed.